app/views/layouts/application.html.haml

```diff
@@ -9,16 +9,17 @@
     %title
       = retrieve_title
     = favicon_link_tag 'favicon.ico'
-    = stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': true
-    = javascript_include_tag 'vendor/modernizr'
-    = javascript_include_tag 'application', 'data-turbolinks-track': true
+    = stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload'
+    // = javascript_include_tag 'vendor/modernizr'
```

Comment:
@asideofcode-dev might as well just delete this line, unless you think we might add it back later for some reason?